src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php

```diff
@@ -103,7 +104,6 @@ public function registerBundles()
 
         if (self::CONTEXT_ADMIN === $this->getContext()) {
             $bundles[] = new \Symfony\Bundle\SecurityBundle\SecurityBundle();
-            $bundles[] = new \Sulu\Bundle\AdminBundle\SuluAdminBundle();
```

Comment:
shouldnt we do this for the other bundles in this block too? 🤔

Reply:
its a step by step, take some work todo for all.